### PR TITLE
Add kops 1.18 to kops test grid

### PR DESCRIPTION
**Pull request by justinsb**

### CODE REVIEW

File: config/jobs/kubernetes/kops/build-grid.py [hunk 1]
```diff
@@ -232,7 +249,7 @@ def expand(s):
     None,
     'calico',
     'cilium',
-    'flannel',
+    'flannl',
```
Comment (mikesplain) [inline config/jobs/kubernetes/kops/build-grid.py:252]: Flannel typo?

Comment (mikesplain) [reply config/jobs/kubernetes/kops/build-grid.py:252]: Ahh nevermined.

File: config/jobs/kubernetes/kops/kops-periodics-grid.yaml [hunk 2]
```diff
@@ -1682,30 +1830,7282 @@ periodics:
...
+      - --kops-args=--networking=cilium
+      - --kops-image=075585003325/Flatcar-stable-2512.2.0-hvm
+      - --kops-priority-path=/workspace/kubernetes/platforms/linux/amd64
+      - --kops-version=https://storage.googleapis.com/kops-ci/markers/release-{kops_version}/latest-ci-updown-green.txt
+      - --provider=aws
+      - --test_args=--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|\[HPA\]|Dashboard|Services.*functioning.*NodePort|Services.*rejected.*endpoints|Services.*affinity
+      - --timeout=60m
+      image: gcr.io/k8s-testimages/kubekins-e2e:v20200730-e636ee2-1.18
+  annotations:
+    testgrid-dashboards: sig-cluster-lifecycle-kops, google-aws, kops-grid, ko
```
Comment (mikesplain) [inline config/jobs/kubernetes/kops/kops-periodics-grid.yaml:8909]: Is this flag a valid kops flag?

**Comment by mikesplain** [comment]
Github seems messed up in this view with my comment... it looks like we're passing `flannl` as an actual networking config to kops? Is that correct? I assumed it was either copy paste error or shortening to make it look better on testgrid 😄

File: config/jobs/kubernetes/kops/kops-periodics-grid.yaml (hunk 2 above)
Comment (justinsb) [reply config/jobs/kubernetes/kops/kops-periodics-grid.yaml:8909]: Doh!  Great point!

We need to keep the cluster name short; I totally forgot this was passed verbatim to kops - will fix!

Nice catch :-)

**Comment by mikesplain** [comment]
Looks good! Thanks!

/lgtm
/approve

**Pull request merged**